prisma/schema.prisma

```diff
@@ -17,10 +17,11 @@ datasource db {
 
 model User {
   id       Int         @id @default(autoincrement())
-  name     String
+  name     String?
```

Comment:
I think we can avoid making it optional, and add the name form the provider, o use it email username as the initial user, in case the provider doesn't not return the name